cmd/up/up.go

```diff
@@ -364,7 +367,10 @@ func (up *upContext) activateLoop() {
 				<-t.C
 			}
 		}
-
+		if up.isRetry && up.hasDevIDChanged(ctx) {
+			up.Exit <- fmt.Errorf("could not connect to dev: another okteto process is modifying it")
```

Comment:
```suggestion
			up.Exit <- fmt.Errorf("session disconnected: there is another `okteto up` session on this container")
```
Could we also show a hint to run `okteto exec`?